po/es.po

```diff
@@ -72,45 +74,47 @@ msgstr "(no disponible)"
 
 #: hardinfo/cpu_util.c:210 hardinfo/cpu_util.c:212
 msgid "Socket"
-msgstr "Soket"
+msgstr "Zócalo"
 
 #: hardinfo/cpu_util.c:215 hardinfo/cpu_util.c:217 modules/devices/gpu.c:149
 #: modules/devices/gpu.c:227
 msgid "Core"
-msgstr "Core"
+msgstr "Núcleo"
 
 #: hardinfo/cpu_util.c:220
+#, fuzzy
 msgid "Book"
 msgstr "Asentamientos"
 
 #: hardinfo/cpu_util.c:222
+#, fuzzy
 msgid "Drawer"
 msgstr "Dibujado"
 
 #: hardinfo/cpu_util.c:228 modules/devices/arm/processor.c:471
 #: modules/devices/x86/processor.c:750
 msgid "Topology"
-msgstr "Topologia"
+msgstr "Topología"
 
 #: hardinfo/cpu_util.c:229
 msgid "ID"
-msgstr "ID"
+msgstr "Identificador"
```

Comment:
Large words may have son seconds effects! please check first! do you recompile with new lang translation?

Reply:
Yes I do. I have tested the translation trying to find every change and found no issues at all. Please, tell me if you spot something broken.

Reply:
OK! good! here!